plugin/db/util/driverutil.go

```diff
@@ -393,6 +393,70 @@ func ExecuteMigration(ctx context.Context, l *zap.Logger, dbType db.Type, driver
 	return insertedID, afterSchemaBuf.String(), nil
 }
 
+func Query(ctx context.Context, db *sql.DB, statement string) ([]interface{}, error) {
+	rows, err := db.QueryContext(ctx, statement)
```

Comment:
We might wanna use a transaction with read-only option [1] with ABORT() instead of QueryContext() because QueryContext() may still run a mutation query [2].

[1]
https://github.com/go-sql-driver/mysql/blob/bcc459a906419e2890a50fc2c99ea6dd927a88f2/connection.go#L471

[2]
INSERT INTO users (firstname, lastname) VALUES ('Joe', 'Cool') RETURNING id;